public/js/apps/payment/app.jsx

```diff
@@ -67,7 +68,10 @@ export default class PaymentApp extends Component {
             } else {
               console.log('rendering purchase flow');
               return (
-                <Purchase user={result.user} productId={state.productId} />
+                <Purchase
+                 productId={state.productId}
```

Comment:
is this indented for vertical alignment? Everywhere else we've been doing two space indents.